### ISSUE TITLE
Single-handed emergency gestures

### ISSUE THREAD

**Issue by LexiBasilisk** [Yellow-Dog-Man/Resonite-Issues#541]
### Is your feature request related to a problem? Please describe.

The emergency gestures (respawning, disconnecting from the world, etc.) are not accessible to those with access to only one hand/controller, or those that are unable to lift their arms up to their head.

### Describe the solution you'd like

If emergency gestures were made accessible to single-handed use, this functionality would be more easily accessed by those that otherwise currently can't do so effectively.

### Describe alternatives you've considered

Alternatively (and possibly additionally), if this functionality were made more readily accessible, e.g. as a facet, it would improve visibility of this functionality/allow those that need it to find and activate it more easily.

### Additional Context

Follow-up from an internal sync where I had raised the topic of accessibility on the platform.

**Comment by Frooxius** [comment]
We'd need to figure out what gestures we could use for this that would work across variety of controllers and be both reliable, but also not triggered by accident. 

I think it might be worthwhile to simply switch out the gesture to a single handed one that doesn't rely on specific pose of the hand. 

Perhaps something like repeated mashing of the menu button? Say you press it rapidly a few times in a row?

**Comment by Stellanora64** [comment]
> Perhaps something like repeated mashing of the menu button? Say you press it rapidly a few times in a row?

Since this feature is meant for accessibility reasons, it would be good to also take into consideration people with motor control disabilities that may not be able to perform rapid movements.

May be as an alternative, a combination press and hold may work? Something like press and hold the trigger and Context menu button for 5 seconds to respawn, and press and hold the trigger and secondary for 5 seconds to disconnect.

**Comment by JackTheFoxOtter** [comment]
> Of note- while researching options that VRChat has for this- while their 'panic button' combination is two-handed, it is also available as a toggle on their menu.

This would be an option, but it wouldn't be much different than just putting the "exit world" facet to the top bar of your dash, which is what I'm doing.

The issue with anything that requires visuals is that emergency gestures are especially useful if you cannot use the visual UI, for example due to floating point precision issues or something blocking your laser. In those cases the feature wouldn't be accessible to single controller users.
